api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirMedicationRequestDaoImpl.java

```diff
@@ -52,6 +95,7 @@ protected void setupSearchParams(Criteria criteria, SearchParameterMap theParams
 					break;
 			}
 		});
+		excludeDiscontinueOrders(criteria);
```

Comment:
The "getSearchResultsUuid" method relied on the search params, so instead of overriding it to filter out discontinue orders, I could modify the criteria directly in the setupSearchParams function.